pkg/controller/seed-controller-manager/mla/dashboard_grafana_controller_test.go

```diff
@@ -26,11 +26,13 @@ import (
 	"testing"
 	"time"
 
-	grafanasdk "github.com/kubermatic/grafanasdk"
 	"github.com/stretchr/testify/assert"
+
+	grafanasdk "github.com/kubermatic/grafanasdk"
```

Comment:
fix import order

Reply:
Isn't it? `grafanasdk` is under Kubermatic organization

Reply:
yes, sorry I've noticed first `github.com`